hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/ErrorMessage.java

```diff
@@ -31,6 +31,7 @@
 
 	private final String errorSequence;
 	private final String errorMessage;
+	private String fieldValue;
```

Comment:
You've added state (fieldValue) to something that is just a stateless enum. You will have multiple threads modifying this enum at the same time which could result in the wrong value being picked up.